avocado/core/jobdata.py

```diff
@@ -0,0 +1,217 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2016
+# Author: Amador Pahim <apahim@redhat.com>
+
+"""
+Record/retrieve job information
+"""
+
+import ast
+import glob
+import os
+import pickle
+import sys
+
+
+from . import exit_codes
+from .settings import settings
+from ..utils.path import init_dir
+
+
+JOB_DATA_DIR = 'jobdata'
+JOB_DATA_FALLBACK_DIR = 'replay'
+CONFIG_FILENAME = 'config'
```

Comment:
I don't think the config/urls/multiplex/... should be defined as variables. They basically copy the variables, making it clear even without definition.